### PR TITLE
feat: expose endpoint to get tenant admins for super tenant admin role

### PR DESCRIPTION
**Pull request by tkuzynow**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: src/main/java/de/caritas/cob/userservice/api/port/out/AdminRepository.java [hunk 1]
```diff
@@ -31,5 +31,8 @@ public interface AdminRepository extends CrudRepository<Admin, String> {
   @Query(value = "SELECT a " + "FROM Admin a " + "WHERE " + "id = ?1 AND type = ?2")
   Optional<Admin> findByIdAndType(String adminId, Admin.AdminType type);
 
+  @Query(value = "SELECT a " + "FROM Admin a " + "WHERE tenantId = ?1 AND type = ?2")
```
Comment (adnanalicic) [inline src/main/java/de/caritas/cob/userservice/api/port/out/AdminRepository.java:34]: why not as a string, inteads of string concatenation?

File: src/main/java/de/caritas/cob/userservice/api/admin/service/admin/TenantAdminUserService.java [hunk 2]
```diff
@@ -98,4 +99,14 @@ public Map<String, Object> findTenantAdminsByInfix(String infix, PageRequest pag
     return userServiceMapper.mapOfAdmin(
         adminsPage, fullAdmins, Lists.newArrayList(), Lists.newArrayList());
   }
+
+  public List<AdminResponseDTO> findTenantAdmins(Long tenantId) {
+    var admins = retrieveAdminService.findTenantAdminsByTenantId(tenantId);
+    if (admins != null) {
```
Comment (adnanalicic) [inline src/main/java/de/caritas/cob/userservice/api/admin/service/admin/TenantAdminUserService.java:105]: are you sure that admins can be null if no entries are found?

**Review by adnanalicic — APPROVED** [review]

File: src/main/java/de/caritas/cob/userservice/api/port/out/AdminRepository.java (hunk 1 above)
Comment (tkuzynow) [reply src/main/java/de/caritas/cob/userservice/api/port/out/AdminRepository.java:34]: fixed

File: src/main/java/de/caritas/cob/userservice/api/admin/service/admin/TenantAdminUserService.java (hunk 2 above)
Comment (tkuzynow) [reply src/main/java/de/caritas/cob/userservice/api/admin/service/admin/TenantAdminUserService.java:105]: they should never be null, you're right. I will remove redundant check.

**Pull request merged**